binder/mybinder_exemplary_dockerfile/README.md

```diff
@@ -0,0 +1,6 @@
+# Dockerfile for mybinder
+To configure mybinder to run the notebooks from your repositories, you can simply include a Dockerfile, like the one in this folder, in the root of your repository.
+## Details of the Dockerfile
```

Comment:
I guess you can place the Readme directly in the binder folder

Reply:
I don't why there still a mybiner_examplary_dockerfile directory! I removed it.